osf_tests/management_commands/test_migrate_notifications.py

```diff
@@ -0,0 +1,216 @@
+import pytest
```

Comment:
@Johnetordoff What about users with partially full digests at time of migration can we time migration to be immediately after those are sent?

Reply:
To answer my own question, because we haven't altered the celery beat queue in all digests in the queue should execute in the old style.